dev/test-create-integration-studio/sanity.config.ts

```diff
@@ -55,5 +55,10 @@ export default defineConfig([
         startInCreateEnabled: false,
       },
     },
+    apps: {
```

Comment:
You need to replace the other create beta config with canvas equivivalents in this file